Jenkinsfile

```diff
@@ -61,9 +126,6 @@ node('vetsgov-general-purpose') {
     }
   }
 
-  // Perform a build for each build type
-  envsUsingDrupalCache = commonStages.buildAll(ref, dockerContainer, params.cmsEnvBuildOverride != 'none')
```

Comment:
Since we're no longer using `buildAll`, can we take it out of `/jenkins/common.groovy`?

Reply:
Theoretically, yes. However, I wanted to limit the potential risk of this PR. So I would suggest trying to remove the `buildAll` from `common.groovy` in a separate PR. 

It's also worth noting that the [`devops` repo references `common.groovy`](https://github.com/department-of-veterans-affairs/devops/search?q=common.groovy), so it's possible the `buildAll` function could be used in other places outside of `vets-website`.